hs_core/tests/api/rest/test_copy_resource.py

```diff
@@ -0,0 +1,37 @@
+import os
+import tempfile
+import zipfile
+
+from rest_framework import status
+
+from hs_core.hydroshare import resource
+from hs_core.tests.api.utils import MyTemporaryUploadedFile
+
+from .base import HSRESTTestCase
+
+
+class TestPublicCopyResourceEndpoint(HSRESTTestCase):
+    def setUp(self):
+        super(TestPublicCopyResourceEndpoint, self).setUp()
+
+        self.rtype = 'GenericResource'
+        self.title = 'My Test resource'
+        res = resource.create_resource(self.rtype,
+                                       self.user,
+                                       self.title,
+                                       unpack_file=False)
+
+        self.pid = res.short_id
+        self.resources_to_delete.append(self.pid)
+
+    def test_copy(self):
+        copy_url = "/hsapi/resource/%s/copy/" % self.pid
+        response = self.client.post(copy_url, {}, format='json')
+        new_pid = response.url.split('/')[4]
+        self.resources_to_delete.append(new_pid)
```

Comment:
Just curious where this self.resources_to_delete list is used to actually delete all created resources. I don't see the resource deletion part from code changes.